ci/glci/alicloud.py

```diff
@@ -134,22 +136,30 @@ def _share_images(self, region_image_map: dict):
         self.acs_client.set_region_id(self.region)
 
     # delete images
-    def delete_images(self, region_image_map: dict):
-        for region, image_id in region_image_map.items():
-            self.acs_client.set_region_id(region)
-            logger.info(
-                f"delete image ({region}/{image_id})"
-            )
-            req = ModifyImageSharePermissionRequest.ModifyImageSharePermissionRequest()
-            req.set_ImageId(image_id)
-            req.set_LaunchPermission(str(ImageShareOption.UNSHARE))
-            self.acs_client.do_action_with_exception(req)
+    def delete_images(self):
 
-            req = DeleteImageRequest.DeleteImageRequest()
-            req.set_ImageId(image_id)
-            self.acs_client.do_action_with_exception(req)
-
-        self.acs_client.set_region_id(self.region)
+        regions = self._list_regions()
+        regions.append(self.region)
+        for region in regions:
+            exist, image_id = self._check_image_existance(region,
```

Comment:
I think it would be a good idea to extract the functionality to remove a single image into an own method